amp/system-oracle/README.md

```diff
@@ -0,0 +1,46 @@
+# System image build with Oracle Database support
+
+
+## Prerequisites
+
+In order to build the image with Oracle Database support,
+you will need to download and add the binaries inside the directory `oracle-client-files`.
```

Comment:
I'd clarify that this is a directory on your local machine where you will have to run the commands further down.